packages/layout/src/components/cards/common/views/preview/components/companiesHouseNumber/companiesHouseNumber.tsx

```diff
@@ -12,7 +12,7 @@ export const CompaniesHouseNumber: React.FC<ICompaniesHouseNumberProps> = React.
 		return (
 			<Flex cfg={{ flexDirection: 'column', mt: 5 }}>
 				<UnderlinedButton>{heading}</UnderlinedButton>
-				<P>{houseNumber}</P>
+				<P cfg={{ mt: 2 }}>{houseNumber}</P>
```

Comment:
If there is not enough space between `UnderlinedButton` and anything that follows it, it's better to add the right amount of space below `UnderlinedButton` by default rather than asking every location it's used to add `margin-top` and expect them to do so consistently.